dependencies.yaml

```diff
@@ -376,6 +379,18 @@ dependencies:
           - matrix:
             packages:
               - python>=3.9,<3.11
+  test_libcuml:
+    common:
+      - output_types: conda
+        packages:
+          - libcuml==24.4.*
+          - libcuml-tests==24.4.*
+  test_cuml:
+    common:
+      - output_types: conda
+        packages:
+          - libcuml==24.4.*
+          - cuml==24.4.*
```

Comment:
Would we want this for wheels too?

Reply:
No, it's not needed. This dependency list is used exclusively for conda CI, to generate the environment with the PR artifacts and test dependencies in one solve. Wheels go through a different installation path and I'm not sure if it's possible to consolidate that environment generation into one step or not. Even if it is possible, it's unlikely that listing `cuml-cu12` in the requirements file would be sufficient, since we need to install a specific filename for the wheel that was fetched from CI artifacts.